README.md

```diff
@@ -2,10 +2,43 @@
 
 [![License](https://img.shields.io/github/license/Qiskit/arraylias.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)
 
-**This repo is unreleased and is still in the early stages of development, there will be breaking API changes.**
+**This repo is still in the early stages of development, there will be breaking API changes**
 
-A Python library for automatic aliasing of multiple array libraries.
+Arraylias is an open-source Python library for writing Python code that
+can be run on multiple array types by aliasing different array libraries.
+Arraylias enables users to reduce the time and effort of writing codes to use various arrays.
+Arraylias provides supports for aliasing some array libraries, [NumPy](https://github.com/numpy/numpy),
+[JAX](https://github.com/google/jax), and [Tensorflow](https://github.com/tensorflow).
+
+Also, Arraylias provides another features about [Scipy](https://github.com/scipy/scipy). 
+Scipy is an open-source Python library for scientific computing. It builds on Numpy and provides support for mathematical operations, matrix calculation, and multi-dimensional arrays. Some array types allows to use scipy-like modules, which are built on the respective array libraries. Arraylias provides aliases for these scipy-like modules.
```

Comment:
Just an edit on wording here. I've tried to restructure it so that the first paragraph describes the general capabilities of Arraylias, and then the second describes the specific default aliases for numpy/scipy.

```suggestion
Arraylias is an open-source Python library providing single-dispatching tools centred around the
construction of an aliased module. Aliased modules are built by initially registering "libraries"
consisting of a collection of types, then registering different versions of a given function in the
aliased module for each underlying type library. When using the aliased module, function calls are
automatically dispatched to version of the function for the correct library based on the type of the
first argument.

Arraylias contains default pre-built aliased versions of both
[NumPy](https://github.com/numpy/numpy) and [Scipy](https://github.com/scipy/scipy), with additional
registration of the [JAX](https://github.com/google/jax) and
[Tensorflow](https://github.com/tensorflow) array libraries. This enables writing
[NumPy](https://github.com/numpy/numpy) and [Scipy](https://github.com/scipy/scipy) like code that
will that will execute on [NumPy](https://github.com/numpy/numpy),
[JAX](https://github.com/google/jax), and [Tensorflow](https://github.com/tensorflow) array objects
as if it had been written in the respective native libraries. If necessary, these default aliases
can be further extended to fit the needs of the application.
```